ccms_integration/ccms_spec.rb

```diff
@@ -2,99 +2,6 @@
-             injunction_recent_incident_detail: '29/03/2019',
-             lead_proceeding_indicator: true,
-             level_of_service: 3,
-             matter_type_meaning: 'Domestic Abuse',
-             meaning: 'Non-molestation order-Domestic Abuse',
-             police_notified?: true,
-             proceeding_level_of_service: 'Full Representation',
-             reason_no_injunction_warning_letter: 'Too dangerous',
-             requested_scope: 'MULTIPLE',
-             scope_limitations: [scope_limitation_1, scope_limitation_2],
-             status: 'draft',
-             warning_letter_sent?: false
-    end
-
-    let(:proceeding_type_2) do
-      double ProceedingType,
-             bail_conditions_set?: false,
-             case_id: 'P_11594793',
-             ccms_category_law: 'Family',
-             ccms_category_law_code: 'MAT',
-             ccms_code: 'SE014',
-             ccms_matter: 'Children-Section 8 orders',
-             ccms_matter_code: 'KSEC8',
-             delegated_functions_date: Date.new(2019, 4, 1),
-             description: 'to be represented on an application for a child arrangements order ?where the child(ren) will live',
-             family_prospects_of_success: '50% or Better',
-             fam_prosp_50_or_better: true,
-             fam_prosp_border_uncert_poor: false,
-             fam_prosp_borderline_uncert: false,
-             fam_prosp_good: false,
-             fam_prosp_marginal: false,
-             fam_prosp_poor: false,
-             fam_prosp_uncertain: false,
-             fam_prosp_very_good: false,
-             fam_prosp_very_poor: false,
-             inj_respondent_capacity?: true,
-             injunction_recent_incident_detail: '29/03/2019',
-             lead_proceeding_indicator: false,
-             level_of_service: 3,
-             matter_type_meaning: 'Section 8 orders',
-             meaning: 'CAO residence',
-             police_notified?: true,
-             proceeding_level_of_service: 'Full Representation',
-             reason_no_injunction_warning_letter: 'Too dangerous',
-             requested_scope: 'MULTIPLE',
-             scope_limitations: [scope_limitation_1, scope_limitation_3],
-             status: 'draft',
-             warning_letter_sent?: false
-    end
-
-    let(:scope_limitation_1) do
-      double 'ScopeLimitation',
-             limitation: 'CV118',
-             wording: 'Limited to all steps up to and including the hearing on 01/04/2019',
-             delegated_functions_apply: true
-    end
-
-    let(:scope_limitation_2) do
-      double 'ScopeLimitation',
-             limitation: 'AA019',
-             wording: scope_limitation_wording_2,
-             delegated_functions_apply: false
-    end
-
-    let(:scope_limitation_3) do
-      double 'ScopeLimitation',
-             limitation: 'FM049',
-             wording: scope_limitation_wording_3,
-             delegated_functions_apply: false
-    end
-
```

Comment:
Great work in getting rid of these doubles.  we can also delete the `scope_limitation_wording_n` methods in this spec as well as they are only referenced from this deleted code.

Reply:
Done 👍